app/lib/jurisdiction_ids_and_paths_cache_invalidator.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# Helper method that will invalidate the hash of all jurisdictions and ids
```

Comment:
I would be in favor (because it has come up before) of just having a new `helper.rb` file here that these sort of things can go in.

Reply:
I could see creating a generalized Module for cache invalidation when/if there are more points we use the cache. I'm not sure about grouping this functionality into a file with other things, though. I don't see the harm in making specific modules.